src/views/collection-view/components/collection-meta-progress/index.tsx

```diff
@@ -0,0 +1,98 @@
+import { useMemo } from 'react'
+import ButtonLink from 'components/button-link'
+import { IconCollections16 } from '@hashicorp/flight-icons/svg-react/collections-16'
+import { IconCheckCircleFill16 } from '@hashicorp/flight-icons/svg-react/check-circle-fill-16'
+import { useCollectionProgress } from 'hooks/progress'
+import { Collection } from 'lib/learn-client/types'
```

Comment:
```suggestion
import { Collection as ClientCollection } from 'lib/learn-client/types'
```
nit: for consistency with other imports of this type

Reply:
✅ updated! ty 💖